src/main/steps/review-pay-submit/payment/pay-your-fee/get.ts

```diff
@@ -30,14 +36,71 @@ export default class PayYourFeeGetController extends GetController {
           if (err) {
             throw err;
           }
-          super.get(req, res);
         });
         return;
       } else {
         throw new Error('Unable to get fee from fee-register API');
       }
     }
+    ////////////////////////////////////////////////////////////////////
+    // const fields = typeof this.fields === 'function' ? this.fields(req.session.userCase) : this.fields;
+    // const form = new Form(fields);
+    // const { _csrf, ...formData } = form.getParsedBody(req.body);
+    // req.session.errors = form.getErrors(formData);
+    //Object.assign(req.session.userCase, formData);
 
-    super.get(req, res);
+    const applicationFeeOrderSummary = req.session.userCase.applicationFeeOrderSummary;
+    //const fee = applicationFeeOrderSummary?.Fees[0]?.value;
+    const fee =
+      applicationFeeOrderSummary && applicationFeeOrderSummary.Fees
+        ? applicationFeeOrderSummary?.Fees[0]?.value
+        : undefined;
+    if (!fee) {
+      req.session.errors?.push({ errorType: 'errorRetrievingFee', propertyName: 'paymentType' });
+      this.redirect(req, res, req.url);
+      return;
+    }
```

Comment:
this code is not required.. because if fee is null then you are throwing error from line 42